setup.py

```diff
@@ -1,23 +1,87 @@
+
+
+def get_requirements(requirements_filename: str):
+    requirements_file = path.join(path.abspath(path.dirname(__file__)),
+                                  requirements_filename)
+    with open(requirements_file, 'r', encoding='utf-8') as r:
+        requirements = r.readlines()
+    requirements = [r.strip() for r in requirements if r.strip()
+                    and not r.strip().startswith("#")]
+    if 'MYCROFT_LOOSE_REQUIREMENTS' in os.environ:
+        print('USING LOOSE REQUIREMENTS!')
+        requirements = [r.replace('==', '>=').replace('~=', '>=') for r in requirements]
+    return requirements
+
+
+def find_resource_files():
+    resource_base_dirs = ("locale", "ui", "vocab", "dialog", "regex", "skill")
+    base_dir = path.dirname(__file__)
+    package_data = ["*.json"]
+    for res in resource_base_dirs:
+        if path.isdir(path.join(base_dir, res)):
+            for (directory, _, files) in walk(path.join(base_dir, res)):
+                if files:
+                    package_data.append(
+                        path.join(directory.replace(base_dir, "").lstrip('/'),
+                                  '*'))
+    return package_data
+
+
+with open("README.md", "r") as f:
+    long_description = f.read()
+
+
+def get_version():
+    """ Find the version of this skill"""
+    version_file = os.path.join(os.path.dirname(__file__), 'version.py')
+    major, minor, build, alpha = (None, None, None, None)
+    with open(version_file) as f:
+        for line in f:
+            if 'VERSION_MAJOR' in line:
+                major = line.split('=')[1].strip()
+            elif 'VERSION_MINOR' in line:
+                minor = line.split('=')[1].strip()
+            elif 'VERSION_BUILD' in line:
+                build = line.split('=')[1].strip()
+            elif 'VERSION_ALPHA' in line:
+                alpha = line.split('=')[1].strip()
+
+            if ((major and minor and build and alpha) or
+                    '# END_VERSION_BLOCK' in line):
+                break
+    version = f"{major}.{minor}.{build}"
+    if int(alpha):
+        version += f"a{alpha}"
+    return version
 
 setup(
     # this is the package name that goes on pip
     name='ovos-skill-tunein',
-    version='0.0.1',
+    version=get_version(),
     description='ovos tunein skill plugin',
-    url='https://github.com/JarbasSkills/skill-tunein',
+    long_description=long_description,
+    url=URL,
     author='JarbasAi',
     author_email='jarbasai@mailfence.com',
     license='Apache-2.0',
-    package_dir={"skill_tunein": ""},
-    package_data={'skill_tunein': ['locale/*', 'ui/*']},
-    packages=['skill_tunein'],
+    package_dir={SKILL_PKG: ""},
+    package_data={SKILL_PKG: find_resource_files()},
+    packages=[SKILL_PKG],
     include_package_data=True,
-    install_requires=["ovos_workshop~=0.0.5a7", "tunein~=0.0.1"],
+    install_requires=get_requirements("requirements.txt"),
```

Comment:
will need the other PR merged first and new tunein package version added here before merging

Reply:
`tunein` is still `0.0.1`, but ovos_workshop could be upped, right?

Reply:
Yes, need a TuneIn release with the new method and to bump workshop to a non alpha